src/stories/containers/TransparencyReport/components/TransparencyTransferRequest/useTransparencyTransferRequest.tsx

```diff
@@ -89,6 +98,44 @@ export const useTransparencyTransferRequest = (currentMonth: DateTime, budgetSta
     return result;
   }, [currentMonth, budgetStatements]);
 
+  const getTransferRequestTargetBalanceColumn = useCallback((wallet: BudgetStatementWalletDto) => {
+    const targetWithTimeSpan: Pick<BudgetStatementWalletTransferRequestDto, 'target' | 'walletBalanceTimeStamp'> =
+      {} as BudgetStatementWalletTransferRequestDto;
+    wallet.budgetStatementTransferRequest?.forEach((item: BudgetStatementWalletTransferRequestDto) => {
+      targetWithTimeSpan.target = item.target;
+      targetWithTimeSpan.walletBalanceTimeStamp = item.walletBalanceTimeStamp;
+    });
```

Comment:
In this `forEach` you're actually setting to `targetWithTimeSpan.target` and `targetWithTimeSpan.walletBalanceTimeStamp` the values of `target` and `walletBalanceTimeStamp` of the last `budgetStatementTransferRequest` item, is that what you actually want to do? if the answer is yes, this code is equivalent to:

```ts
const lastIndex = wallet.budgetStatementTransferRequest?.length - 1;
targetWithTimeSpan.target = wallet.budgetStatementTransferRequest?[lastIndex]?.target;
targetWithTimeSpan.walletBalanceTimeStamp = wallet.budgetStatementTransferRequest?[lastIndex]?.walletBalanceTimeStamp;
```

Reply:
You are right, in fact, this is only one item in the budgetStatementTransferRequest, so I got your point, and your approach

Reply:
This has not been fixed yet

Reply:
my bad, I take you approach but don't delete the old code